embedchain/pipeline.py

```diff
@@ -346,22 +342,40 @@ def deploy(self):
         self.telemetry.capture(event_name="deploy", properties=self._telemetry_props)
 
     @classmethod
-    def from_config(cls, yaml_path: str, auto_deploy: bool = False):
+    def from_config(cls, yaml_path: Optional[str] = None, auto_deploy: bool = False, **kwargs: Dict[str, Any]):
```

Comment:
Let's not pass json_path and config in the kwargs. I think we can list them down directly as args with default value of None